.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+
+Fixes #  
+
+### Changes proposed in this pull request: 
+
+ - 
+ - 
```

Comment:
yeah great suggestions @ColinSullivan1 

Have used `- [ ] Build is green` too as a double check besides the result from Travis in case want to consider (just so that it is explicit)